build.sbt

```diff
@@ -162,33 +144,30 @@ ThisBuild / mimaBinaryIssueFilters ++= Seq(
     "fs2.io.net.SocketCompanionPlatform$IntCallbackHandler"
   ),
   ProblemFilters.exclude[MissingClassProblem]("fs2.Chunk$BufferChunk"),
-  ProblemFilters.exclude[DirectMissingMethodProblem]("fs2.Chunk.makeArrayBuilder")
+  ProblemFilters.exclude[DirectMissingMethodProblem]("fs2.Chunk.makeArrayBuilder"),
+  ProblemFilters.exclude[DirectMissingMethodProblem](
+    "fs2.interop.reactivestreams.StreamSubscriber.fsm"
+  ),
+  ProblemFilters.exclude[Problem]("fs2.interop.reactivestreams.StreamSubscriber#FSM*"),
+  ProblemFilters.exclude[DirectMissingMethodProblem]("fs2.io.package.utf8Charset"),
+  ProblemFilters.exclude[DirectMissingMethodProblem](
+    "fs2.io.net.tls.TLSSocketPlatform.applicationProtocol"
+  ),
+  ProblemFilters.exclude[DirectMissingMethodProblem]("fs2.compression.Compression.gzip$default$*"),
+  ProblemFilters.exclude[DirectMissingMethodProblem]("fs2.compression.Compression.gunzip$default$1$")
```

Comment:
I have some optimism that these missing methods will be restored once the `gzip` and `gunzip` methods move back to the shared `Compression` trait in https://github.com/typelevel/fs2/pull/2771.